tests/benchdnn/CMakeLists.txt

```diff
@@ -165,6 +168,17 @@ foreach(driver ${all_drivers})
         inputs/brgemm/test_brgemm_all inputs/conv/test_conv_all
         inputs/graph/test_graph_all inputs/rnn/test_lstm_all)
 
+    if(PYTHON3_FOUND AND ${driver} IN_LIST RANDOM_CI_DRIVERS)
+        set(SYNTHDNN "${CMAKE_CURRENT_SOURCE_DIR}/../synthdnn/synthdnn.py")
+        set(RANDOM_CI "${CMAKE_CURRENT_BINARY_DIR}/inputs/${driver}/test_${driver}_random_ci")
+        execute_process(COMMAND ${PYTHON3_EXECUTABLE} ${SYNTHDNN} ${driver} -s400 -b ${RANDOM_CI} RESULT_VARIABLE RET OUTPUT_VARIABLE OUT ERROR_VARIABLE OUT)
+        if(${RET})
+            message(Warn "Generation of ${RANDOM_CI} exited with error code ${RET}")
+        else()
+            set(test_files_ci "${test_files_ci};test_${driver}_random_ci")
+        endif()
+    endif()
+
```

Comment:
I proposed to keep the process of (re-)generating problems manual, so this change shouldn't be needed.

Reply:
Agreed.